facefusion/jobs/job_runner.py

```diff
@@ -86,6 +88,10 @@ def finalize_steps(job_id : str) -> bool:
 			for temp_output_path in temp_output_paths:
 				if not move_file(temp_output_path, output_path):
 					return False
+		else:
```

Comment:
please avoid else in the codebase, at the moment there might be just one action happening but in the future it could trigger anything unexpected